orc8r/tools/fab/dev_utils.py

```diff
@@ -192,6 +198,70 @@ def get_hardware_id_from_vagrant(vm_name: str) -> str:
     return str(hardware_id)
 
 
+def get_hardware_id_from_docker(
+    location_docker_compose: str =
+    '../../../lte/gateway/python/integ_tests/federated_tests/docker/',
+) -> str:
+    """
+    Get the magmad hardware ID from magma running on docker from Vagrant
+    Args:
+        location_docker_compose: location of docker compose used to run FEG
+        by default lte/gateway/python/integ_tests/federated_tests/docker
+    Returns:
+        Hardware snowflake from the VM
+    """
+    with lcd('docker'), hide('output', 'running', 'warnings'), \
+            cd(location_docker_compose):
+        hardware_id = local(
+            'docker-compose exec magmad bash -c "cat /etc/snowflake"',
+        capture=True,
+        )
+    return str(hardware_id)
+
+
+def delete_agw_certs(vm_name: str):
+    """
+    Delete certificates and gw_challenge from AGW
+
+    Args:
+        vm_name: Name of the vagrant machine to use
+    """
+    with settings(warn_only=True), hide('output', 'running', 'warnings'), \
+            cd('/var/opt/magma/certs'):
+        vagrant.setup_env_vagrant(vm_name)
+        sudo('rm gateway.*')
+        sudo('rm gw_challenge.key')
+
+
+def delete_feg_certs(location_docker_compose: str):
+    """
+    Get the magmad hardware ID from magma running on docker from Vagrant
+
+    Args:
+        location_docker_compose: location of docker compose used to run FEG
+        by default lte/gateway/python/integ_tests/federated_tests/docker
```

Comment:
Copied?

Reply:
almost. On an initial version I was running docker inside a Vagrant VM for FEG because I was getting some issues with ports.

Once I fixed I changed all the names, but forgot this description. Thanks for finding it!

Reply:
my bad, yes copy issue